README.md

```diff
@@ -1,33 +1,33 @@
 # Jesse Showalter Website Project Starter Kit
 
-This is how I start my project, maybe you can use it too :) This  starter file is a collectiopn of things I have learned over the years, code snippets I have created and other that were collected from other people who are smarter than me to make soimething I really enjoy using.
```

Comment:
Thanks for leaving low-hanging fruit for us newbies, this is my first pull request.